ssg/constants.py

```diff
@@ -255,6 +257,9 @@
     pcidss=Reference(
         id="pcidss", name=REF_PREFIX_MAP["pcidss"], url="",
         regex_with_groups=r"Req-(\d+)(?:\.(\w+)(?:\.(\w+)(?:\.(\w+))?)?)?"),
+    pcidss4=Reference(
+        id="pcidss4", name=REF_PREFIX_MAP["pcidss4"], url="",
+        regex_with_groups=r"Req-(\d+)(?:\.(\w+)(?:\.(\w+)(?:\.(\w+))?)?)?"),
```

Comment:
```suggestion
        regex_with_groups=r"(\d+)(?:\.(\w+)(?:\.(\w+)(?:\.(\w+))?)?)?"),
```

Since we are dropping the `Req` prefix, I think this doesn't make sense anymore, right?